### PR TITLE
Add Chinese triggers for e5s-e7s

### PR DESCRIPTION
**Pull request by EBlaster**

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e6s.js [hunk 1]
```diff
@@ -24,12 +24,12 @@
         if (data.me == matches.target) {
           return {
             en: 'Enumeration on YOU',
-            fr: 'Enumération sur VOUS',
+            cn: '蓝圈分摊点你',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e6s.js]: You're removing the french translations in this file.  Could you undo that?

File: ui/raidboss/raidboss.css [hunk 2]
```diff
@@ -12,9 +12,9 @@
   position: absolute;
   top: 0;
   left: 0;
-  width: 220px;
-  height: 120px;
-  font-size: 14px;
+  width: 400px;
```
Comment (quisquous) [inline ui/raidboss/raidboss.css]: Did you mean to commit these changes too? Can you remove the CSS changes?

File: ui/raidboss/data/05-shb/raid/e6s.js (hunk 1 above)
Comment (EBlaster) [reply ui/raidboss/data/05-shb/raid/e6s.js]: I was modifying the previous version, sorry! Should be fixed now.

**Comment by quisquous** [comment]
Looks good, thanks!  Can you fix the `npm run lint` errors, possibly by running `npm run lintfix`?

https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#build-problems

File: .travis.yml [hunk 3]
```diff
@@ -23,3 +23,4 @@ matrix:
     script:
       - pylint **/*.py --errors-only
       - black **/*.py --line-length 100 --diff
+
```
Comment (quisquous) [inline .travis.yml]: Can you remove the modification to this file?

File: ui/raidboss/data/05-shb/raid/e6s.js (hunk 1 above)
Comment (Bluefissure) [reply ui/raidboss/data/05-shb/raid/e6s.js]: I think '点你' is a direct and weird translation, maybe '点名' is better, which is the same as all similar triggers in previously translated js files.

**Comment by quisquous** [comment]
Thanks for the translations! Sorry for all the comments on the small stuff, I just don't like unnecessary changes to other files if they don't need to be there.

**Pull request merged**